sormas-backend/src/main/java/de/symeda/sormas/backend/common/CronService.java

```diff
@@ -189,8 +189,13 @@ public void syncInfraWithCentral() {
 		centralInfraSyncFacade.syncAll();
 	}
 
-	@Schedule(hour = "1", minute =  "55", persistent = false)
-	public void deleteExpiredEntities(){
+	@Schedule(hour = "1", minute = "55", persistent = false)
```

Comment:
I think we should have 10-15 minutes in between those two. Maybe state permanent deletion at 2:10